src/main/java/org/folio/rest/impl/EholdingsResourcesImpl.java

```diff
@@ -164,7 +170,45 @@ public void getEholdingsResourcesByResourceId(String resourceId, String include,
   @Override
   public void putEholdingsResourcesByResourceId(String resourceId, String contentType, ResourcePutRequest entity,
       Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    asyncResultHandler.handle(Future.succeededFuture(PutEholdingsResourcesByResourceIdResponse.status(Response.Status.NOT_IMPLEMENTED).build()));
+    ResourceId parsedResourceId = idParser.parseResourceId(resourceId);
+    headerValidator.validate(okapiHeaders);
+    MutableObject<RMAPIService> rmapiService = new MutableObject<>();
+    CompletableFuture.completedFuture(null)
+      .thenCompose(o -> configurationService.retrieveConfiguration(new OkapiData(okapiHeaders), vertxContext))
+      .thenCompose(rmapiConfiguration -> {
+        rmapiService.setValue(new RMAPIService(rmapiConfiguration.getCustomerId(),
+          rmapiConfiguration.getAPIKey(), rmapiConfiguration.getUrl(), vertxContext.owner()));
+        return rmapiService.getValue().retrieveResource(parsedResourceId, null);
+      })
+      .thenCompose(resourceData -> {
+        ResourcePut resourcePutBody;
+        boolean isTitleCustom = resourceData.getTitle().getIsTitleCustom().booleanValue();
```

Comment:
.booleanValue() is unnecessary here and in one other place, Boolean is unboxed to boolean automatically